aten/src/ATen/native/native_functions.yaml

```diff
@@ -185,6 +185,15 @@
   dispatch:
     CompositeExplicitAutograd: sym_constrain_range
 
+
+- func: _same_storage(Tensor(a!) self, Tensor b) -> bool
+
+- func: _data_ptr_allocated(Tensor(a!) self) -> bool
+
+- func: _same_storage_size(Tensor(a!) self, int size) -> bool
```

Comment:
Why not just compare the size to the number of elements? Or divide the size by the elem size in python?

Reply:
I dont want to teach dynamo about typed/untyped storage